tests/toolchain/abigen-pass/singleton_contract.cpp

```diff
@@ -2,9 +2,27 @@
 #include <eosio/eosio.hpp>
 #include <eosio/name.hpp>
 #include <eosio/singleton.hpp>
+
+#include "exclude_from_abi.hpp"
  
 using namespace eosio;
- 
+
+struct [[eosio::table]] out_of_class2 {
```

Comment:
Just curious why those digits (`2`, `5`, `51`...) are at the end of the names?

Reply:
`eosio::name` can't have numbers more than 5. I added 55x into hpp so it is easy to reuse lower numbers in cpp. i.e. less likely names will be collided. To some extent it is random, I may just copy pasted and removed some of 5-s